recipes-qt/automotive/qtapplicationmanager_git.bbappend

```diff
@@ -0,0 +1,30 @@
+#SRC_URI += "https://github.com/qt/qtapplicationmanager.git;branch=5.8"
```

Comment:
If you did not already do so, please remove the line below as well.
 `#SRCREV = "23a0e5a929b5b03f91ae532a765e2b7a543c3a99"`